core/src/apps/debug/show_text.py

```diff
@@ -1,8 +1,9 @@
 import trezor.messages.DebugLinkShowTextStyle as S
+import trezor.ui.components.common.text as text
```

Comment:
Seems like this should be `components.tt.text` (that's what we use in the rest of the codebase) - but let's wait for @matejcik - he knows more than me on this topic

```suggestion
import trezor.ui.components.tt.text as text
```

but I